tests/tests/src/DictTests.mjs

```diff
@@ -43,40 +42,76 @@ let PatternMatching = {
 };
 
 let dict = {
-  key1: 1,
+  key1: false,
   key2: undefined
 };
 
-if (Stdlib_Dict.has(dict, "key1") !== true) {
+if (!("key1" in dict)) {
   throw {
     RE_EXN_ID: "Assert_failure",
     _1: [
       "DictTests.res",
-      43,
+      44,
       2
     ],
     Error: new Error()
   };
 }
 
-if (Stdlib_Dict.has(dict, "key2") !== true) {
+if (!("key2" in dict)) {
   throw {
     RE_EXN_ID: "Assert_failure",
     _1: [
       "DictTests.res",
-      44,
+      46,
+      2
+    ],
+    Error: new Error()
+  };
+}
+
+if ("key3" in dict !== false) {
+  throw {
+    RE_EXN_ID: "Assert_failure",
+    _1: [
+      "DictTests.res",
+      48,
+      2
+    ],
+    Error: new Error()
+  };
+}
+
+if (Object.hasOwn(dict, "toString") !== false) {
```

Comment:
I think  using`hasOwnProperty` or pritmitive runtime is better to us. Introducing `Object.hasOwn` would significantly increase the target requirement.